File: app/src/main/java/ru/tebloev/kotlinactormodel/MainActivity.java
package ru.tebloev.kotlinactormodel;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import akka.actor.Actor;
import akka.actor.ActorRef;
import akka.actor.Props;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ActorRef myActor = App.provideActorSystem().actorOf(Props.create(UiActor.class, this).withDispatcher("main-thread-dispatcher"), "actor");
        myActor.tell("yes, yes, yes", ActorRef.noSender());
    }
}

File: app/src/main/java/ru/tebloev/kotlinactormodel/MainActivity.kt
package ru.tebloev.kotlinactormodel

import akka.actor.ActorRef
import akka.actor.ActorSystem
import akka.actor.Props
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import java.util.concurrent.ExecutorService

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val configurator = CustomExecutorServiceConfigurator(null, null)
        val executorServiceFactory = configurator.createExecutorServiceFactory("id", MainThreadFactory())
        val executorService = executorServiceFactory.createExecutorService()
        val system = ActorSystem.create("MySystem")
        val myActor = system.actorOf(Props.create(UiActor::class.java, this),"actor")
        myActor.tell("test", ActorRef.noSender())

    }
}
